docs/product-installation-guide/src/main/asciidoc/eap-business-central-install-proc.adoc

```diff
@@ -10,16 +10,14 @@ Business Central is a web console that enables you to perform the following task
 .Prerequisites
 * {EAP} 7.0.7 installed on your hard drive. The base directory of the EAP installation is referred to as `__EAP_HOME__`.
 * The following file, available from the site that contains this document:
-** Business Central:
-ifdef::BPMS[`jboss-bpmsuite-7.0.0.Beta02-business-central-eap7.zip`]
-ifdef::BRMS[`jboss-brms-7.0.0.Beta02-business-central-eap7.zip`]
++
```

Comment:
Is the ifdef for BRMS necessary, since this chapter is about BPM Suite on EAP 7? (Not BRMS.) If it's meant to go both ways, then we should probably ifdef the chapter title as well. But this is not the only BPM-specific chapter either.